GameProject/Assets/OpenWorld Assets/Scripts/trdprsnCamera.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class trdprsnCamera : MonoBehaviour
```

Comment:
This class name breaks the naming convention for classes. Please rectify. Also this name is double dutch, the hell is `trdprsn`?

Reply:
Turd Person, or Irish for "Third Person"